src/App.js

```diff
@@ -1,180 +1,221 @@
-import './App.css';
-import styled from 'styled-components'
-import React, { useReducer } from 'react'
-import { Activity, activityReducer } from './activity/Activity';
-import { Button } from './components/Theme'
+import "./App.css";
```

Comment:
Can you fork this repo ? Don't want to merge real activities into the main sandbox repo.

Reply:
You do the forking so it's in your name. I'll submit the PR to the new repo.